static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js

```diff
@@ -20,23 +22,42 @@ define([
     svgs
 ) {
     function init() {
-        var alreadyVisted = storage.local.get('alreadyVisited') || 0,
-            adblockLink = 'https://membership.theguardian.com?INTCMP=adb-mv';
+        var alreadyVisted = storage.local.get('alreadyVisited') || 0;
```

Comment:
Nextgen's JavaScript has a convention of only using a single variable declaration per function.